prow/tide/tide.go

```diff
@@ -49,8 +49,12 @@ import (
 	"k8s.io/test-infra/prow/tide/history"
 )
 
-// For mocking out sleep during unit tests.
-var sleep = time.Sleep
+var (
+	// For mocking out sleep during unit tests.
+	sleep = time.Sleep
+
+	mergeFailureComment = "attempt to merge PR failed. Tide may retry depending on the type of failure"
```

Comment:
It looks like we would end up commenting this string verbatim, so it should start with a capital letter and end with dot.

Reply:
Thank you for the review. Fixed.